src/scripts/visualizations/tooltip.js

```diff
@@ -112,6 +124,12 @@
             changeOpacity.call(this, 0, this.options.tooltip.hideTime);
         };
 
+        var onMouseMove = function (d) {
+            if (this.options.tooltip.followCursor) {
+                show.call(this, d)
```

Comment:
It looks like it'll have the right `this` even without call anyway - unless d3/someone else is hijacking it. Minor comment though, ignore if it doesn't work without call.